src/common/ui/code.vue

```diff
@@ -52,7 +52,7 @@
 <script>
 import 'codemirror/lib/codemirror.css';
 import 'codemirror/theme/eclipse.css';
-import 'codemirror/mode/javascript/javascript';
+import '#/common/ui/code_extras/mode/javascript-mixed/javascript-mixed';
```

Comment:
We usually group all `#` imports at the top.

Reply:
Ah, and `code_extras` is using the wrong convention, it should be `-`, but I'd like to simplify the path e.g. maybe `#/common/ui/code/codemirror-js-mixed`

Reply:
done